### PR TITLE
sync zh-trans content/zh/docs/concepts/workloads/controllers/garbage-…

### PR DESCRIPTION
**Pull request by tanjunchen**
sync  zh-trans  content/zh/docs/concepts/workloads/controllers/garbage-collection.md

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -162,22 +167,8 @@ unauthorized dependents cannot delay deletion of an owner object.
...
- * 对象的 `metadata.finalizers` 字段包含了值 `foregroundDeletion`。
-
-一旦对象被设置为 `deletion in progress` 状态，垃圾收集器会删除对象的所有附属。
- 垃圾收集器在删除了所有 `Blocking` 状态的附属（对象的 `ownerReference.blockOwnerDeletion=true`）之后，它会删除拥有者对象。
-
-
-
 注意，在 `foregroundDeletion` 模式下，只有设置了 `ownerReference.blockOwnerDeletion` 值的附属者才能阻止删除拥有者对象。
-在 Kubernetes 1.7 版本中将增加准入控制器（Admission Controller），基于拥有者对象上的删除权限来控制用户去设置 `blockOwnerDeletion` 的值为 true，所以未授权的附属者不能够延迟拥有者对象的删除。
+在 Kubernetes 1.7 版本中将增加 [admission controller](/docs/reference/access-authn-authz/admission-controllers/#ownerreferencespermissionenforcement)，基于拥有者对象上的删除权限来控制用户去设置 `blockOwnerDeletion` 的值为 true，所以未授权的附属者不能够延迟拥有者对象的删除。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md]:
```suggestion
在 Kubernetes 1.7 版本中将增加 [准入控制器](/docs/reference/access-authn-authz/admission-controllers/#ownerreferencespermissionenforcement)，基于拥有者对象上的删除权限来控制用户去设置 `blockOwnerDeletion` 的值为 true，所以未授权的附属者不能够延迟拥有者对象的删除。
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 2]
```diff
@@ -230,7 +223,7 @@ curl -X DELETE localhost:8080/apis/apps/v1/namespaces/default/replicasets/my-rep
 Here's an example that deletes dependents in foreground:
 -->
 
-下面是一个在前台删除附属对象的例子：
+下面是一个在 `Foreground` 中删除附属对象的示例：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md]:
```suggestion
下面是一个在前台删除附属对象的示例：
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 3]
```diff
@@ -243,7 +236,7 @@ curl -X DELETE localhost:8080/apis/apps/v1/namespaces/default/replicasets/my-rep
 Here's an example that orphans dependents:
 -->
 
-这里是一个孤儿附属的例子：
+这里是一个 `Orphan` 附属的示例：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md]:
```suggestion
这里是一个孤立附属对象的示例：
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md (hunk 2 above)
Comment (tanjunchen) [reply content/zh/docs/concepts/workloads/controllers/garbage-collection.md]: see https://github.com/kubernetes/website/pull/18865/files#diff-563c4c44553d361a4d472a93dd458b7bR205

I think it ’s better not to translate here

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md (hunk 3 above)
Comment (tanjunchen) [reply content/zh/docs/concepts/workloads/controllers/garbage-collection.md]: ditto

**Comment by tanjunchen** [comment]
/cc @tengqm 
done for others

**Pull request merged**